radiography/src/main/java/radiography/ScanScopes.kt

```diff
@@ -0,0 +1,73 @@
+package radiography
+
+import android.view.View
+import radiography.ScannableView.AndroidView
+import radiography.ScannableView.ComposeView
+import radiography.compose.ExperimentalRadiographyComposeApi
+import radiography.compose.composeRenderingError
+import radiography.compose.findTestTags
+import radiography.compose.isComposeAvailable
+
+object ScanScopes {
+
+  @JvmField
+  internal val EmptyScope: ScanScope = ScanScope { emptyList() }
+
+  /** Scans all the windows owned by the app. */
+  @JvmField
+  val AllWindowsScope: ScanScope = ScanScope {
+    WindowScanner.findAllRootViews()
+        .map(::AndroidView)
+  }
+
+  /** Scans the window that currently has focus. */
+  @JvmField
+  val FocusedWindowScope: ScanScope = ScanScope {
+    AllWindowsScope.findRoots()
+        .filterIsInstance<AndroidView>()
+        .filter { it.view.parent?.parent != null || it.view.hasWindowFocus() }
+  }
+
+  /** Scans the given [rootView]. */
+  @JvmStatic
+  fun singleViewScope(rootView: View): ScanScope = ScanScope {
+    listOf(AndroidView(rootView))
+  }
+
+  /**
+   * Scans all Composables in all windows that have the `Modifier.testTag` modifier with the given
+   * [testTag].
+   */
+  @ExperimentalRadiographyComposeApi
+  @JvmStatic
+  @JvmOverloads
+  fun composeTestTagScope(
```

Comment:
Might want to explain what a TestTag is and even link to the doc. https://developer.android.com/reference/kotlin/androidx/compose/ui/platform/package-summary#(androidx.compose.ui.Modifier).testTag(kotlin.String)

> Applies a tag to allow modified element to be found in tests.

It's not clear how this relates to Radiography.

Reply:
filed https://github.com/square/radiography/issues/72

Reply:
Good catch, I linked to the docs in the `ViewFilter` for test tags, but forgot to do so here.